.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     - name: make
       # Fail build if there are warnings
       # build with TLS just for compilation coverage
-      run: make REDIS_CFLAGS='-Werror' BUILD_TLS=yes
+      run: make REDIS_CFLAGS='-Werror -DDEBUG_ASSERTIONS' BUILD_TLS=yes
```

Comment:
I was initially thinking this made sense, but practically we should probably apply it to all of the daily.yml tests as well.

Reply:
That makes sense, I have applied to all the jobs in the daily.yml file too.